tests/functional/heketi/test_heketi_lvm_wrapper.py

```diff
@@ -275,3 +276,44 @@ def test_docker_service_restart(self):
 
         # Create PVC after docker restart
         self.create_and_wait_for_pvcs()
+
+    @pytest.mark.tier2
+    def test_udev_usage_in_container(self):
+        """Validate LVM inside container does not use udev"""
+
+        # Skip the TC if independent mode deployment
+        if not self.is_containerized_gluster():
+            self.skipTest(
+                "Skipping this test case as it needs to run on "
+                "converged mode deployment")
+
+        h_client, h_url = self.heketi_client_node, self.heketi_server_url
+
+        # Create file volume
+        vol_info = heketi_ops.heketi_volume_create(
+            h_client, h_url, self.volume_size, json=True)
+        self.addCleanup(
+            heketi_ops.heketi_volume_delete, h_client, h_url,
+            vol_info.get("id"))
+
+        # Create block volume
+        block_vol_info = heketi_ops.heketi_blockvolume_create(
+            h_client, h_url, self.volume_size, json=True)
+        self.addCleanup(
+            heketi_ops.heketi_blockvolume_delete, h_client, h_url,
+            block_vol_info.get("id"))
+
+        # Check dmeventd service in container
+        err_msg = "dmeventd.service is running on setup"
+        with self.assertRaises(exceptions.ExecutionError, msg=err_msg):
+            openshift_ops.cmd_run_on_gluster_pod_or_node(
```

Comment:
This TC will be skipped in case cluster is indpedent mode, I would suggest to use function `oc_rsh` instead of `cmd_run_on_gluster_pod_or_node`

Reply:
Done